docs-md/tutorials/etl.md

```diff
@@ -0,0 +1,614 @@
+
+And likewise the `shipments`:
+
+```javascript
+db.createCollection("shipments")
+```
+
+Populate the `orders` collection with some initial data. Notice that the `customer_id` references identifiers that we created in our Postgres customers table:
+
+```javascript
+db.orders.insert({"customer_id": "2", "order_id": "13", "price": 50.50, "currency": "usd", "ts": "2020-04-03T11:20:00"})
+db.orders.insert({"customer_id": "7", "order_id": "29", "price": 15.00, "currency": "aud", "ts": "2020-04-02T12:36:00"})
+db.orders.insert({"customer_id": "5", "order_id": "17", "price": 25.25, "currency": "eur", "ts": "2020-04-02T17:22:00"})
+db.orders.insert({"customer_id": "5", "order_id": "15", "price": 13.75, "currency": "usd", "ts": "2020-04-03T02:55:00"})
+db.orders.insert({"customer_id": "7", "order_id": "22", "price": 29.71, "currency": "aud", "ts": "2020-04-04T00:12:00"})
+```
+
+Do the same for shipments. Notice that the `order_id` references order ids we created in the previous collection.
+
+```javascript
+db.shipments.insert({"order_id": "17", "shipment_id": "75", "origin": "texas", "ts": "2020-04-04T19:20:00"})
+db.shipments.insert({"order_id": "22", "shipment_id": "71", "origin": "iowa", "ts": "2020-04-04T12:25:00"})
+db.shipments.insert({"order_id": "29", "shipment_id": "89", "origin": "california", "ts": "2020-04-05T13:21:00"})
+db.shipments.insert({"order_id": "13", "shipment_id": "92", "origin": "maine", "ts": "2020-04-04T06:13:00"})
+db.shipments.insert({"order_id": "15", "shipment_id": "95", "origin": "florida", "ts": "2020-04-04T01:13:00"})
+```
+
+### Start the Postgres and MongoDB Debezium source connectors
+
+With all of our seed data in place, we can process it with ksqlDB. Connect to ksqlDB's server using its interactive CLI. Run the following from your host:
+
+```
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+Before we issue more commands, instruct ksqlDB to start all queries from earliest point in each topic:
+
+```
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Now we can connect ask Debezium to stream Postgres' changelog into Kafka. Invoke the following command in ksqlDB. This creates a Debezium source connector and writes all of its changes to Kafka topics:
+
+```sql
+CREATE SOURCE CONNECTOR customers_reader WITH (
+    'connector.class' = 'io.debezium.connector.postgresql.PostgresConnector',
+    'database.hostname' = 'postgres',
+    'database.port' = '5432',
+    'database.user' = 'postgres-user',
+    'database.password' = 'postgres-pw',
+    'database.dbname' = 'customers',
+    'database.server.name' = 'customers',
+    'database.history.kafka.bootstrap.servers' = 'broker:9092',
+    'database.history.kafka.topic' = 'customers',
+    'table.whitelist' = 'public.customers',
+    'transforms' = 'unwrap',
+    'transforms.unwrap.type' = 'io.debezium.transforms.ExtractNewRecordState',
+    'transforms.unwrap.drop.tombstones' = 'false',
+    'transforms.unwrap.delete.handling.mode' = 'rewrite'
+);
+```
+
```

Comment:
You might want to add a "Your output should resemble:" block.

Reply:
Done, thanks!